test/functionalTest/cases/0000_deprecated_checkings/get_entity_dates_with_options.test

```diff
@@ -135,13 +135,13 @@ echo
 
 echo "06. Check dateCreate == t_0 and dateModified == t_1"
 echo "==================================================="
-if [ "$CRE_DATE" == "$T_0" ]
+if [[ "$CRE_DATE" == "$T_0"* ]]
```

Comment:
I think it would be clearer to keep this (and other) ifs as they were, removing the milis part at CRE_DATE setting date. Something like this:

```
# We remove miliseconds from variables as the reference T_0 and T_1 dates doesn't include them
CRE_DATE=$(echo "$_response" | python -mjson.tool | grep dateCreated | awk -F '"' '{print $4}' | awk -F '.' '{print$2}')
MOD_DATE=$(echo "$_response" | python -mjson.tool | grep dateModified | awk -F '"' '{print $4}' | awk -F '.' '{print$2}')
```

Similar comment applicable to other .test (test/functionalTest/cases/0876_attribute_dates/attribute_dates.test and not use if others9

Reply:
Discussed with @kzangeli at skype, this will be done in a next PR. @fgalan will do it. NTC

Reply:
Perhaps ... perhaps this works:
```
if [[ "$CRE_DATE" == "$T_0.???Z" ]]
```

Worth a try ...

Reply:
I'll try that.

Reply:
Tested. I'm afraid it doesn't work.

Reply:
Pity.
Frankly, I believe the current solution is pretty good, only that is doesn't make sure there are exactly three decimals and also that the trailing 'Z' is present.

If you want it "even better", I'm sure this can be solved using regex in bash though.
It seems the "=~" operator is used.
Found a SOF question about something similar: https://stackoverflow.com/questions/18709962/regex-matching-in-a-bash-if-statement/18710850.

Reply:
I found a way:

```
dt="2020-07-03T09:24:04"
dtWithMillis="$dt.456Z"
echo dt: $dt
echo dtWithMillis: $dtWithMillis


if [[ "$dtWithMillis" =~ $dt.[0-9]{3}Z ]]
then
  echo MATCH
else
  echo NO MATCH
fi
```

Reply:
Even in the case comparison condition can be changed, the statement in case of fail

```
   echo "dateCreated: not t_0 ($CRE_DATE != $T_0)"
```

would be a bit weird, as CRE_DATE and T_0 are not sharing a common reference format. So, I think that using the "mínimo común denominador" (I don't know right now how is in English :) of both formats is better. I mean, in this case, to take the one used by `date` as a minimum and "tail" part of the other.

Test modified according to this idea in PR https://github.com/telefonicaid/fiware-orion/pull/3672